### DISCUSSION TITLE
Hack Sonoff Zbbridge-pro and Esphome - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#28] [Comments]
# Hack Sonoff Zbbridge-pro and Esphome - Haade.fr

How to easily hack the Sonoff Zbbridge-pro easily on Esphome and Home-Assistant and make it a Zigbee, wifi and Bluetooth remote control

https://haade.fr/en/blog/zbbridge-pro-flasher-esphome-home-assistant

**Comment by PasekJ** [comment]
Great article!
I really like that ESPhome is used. I have everything set up on ESPhome, but even though you have described everything perfectly, I could not connect the coordinator to HA. I've tried both methods with the integration of ZHA and Zigbee2MQTT. I have two test Sonoff ZBbridge-pro one with SonoffZBPro_coord_20220219.hex and the other with latest fw SonoffZBPro_coord_20230507.hex. If I use fw Tasmota everything works. I'm sorry I can't revive it.
Good luck with your work.
